src/parser/Parser.ts

```diff
@@ -1415,6 +1416,9 @@ export class Parser {
             while (true) {
                 if (match(Lexeme.LeftParen)) {
                     expr = finishCall(expr);
+                } else if (match(Lexeme.Print)) {
```

Comment:
We might need to do a more thorough check here, because the literal `print` isn't supported, but `?` IS. So perhaps `peek().text === '?'` instead?

Reply:
why we need to support literal `print`?

Reply:
We don't. I'm saying the opposite. I was assuming that tokens of kind `Lexeme.print` would contain either `"?"` OR `"print"` text. However, after looking at the lexer, it can only ever contain `?`, so this is a non-issue. (I was getting the brighterscript parser and this parser mixed up, my bad!)

Reply:
ok, sure